### PR TITLE
Instanciate VC context based on issuer domain

### PR DESCRIPTION
**Pull request by NSeydoux**
According to the [ESS documentation](https://docs.inrupt.com/ess/latest/services/service-vc/#ess-vc-service-endpoints), the JON-LD context for issued VCs will depend on the issuer's domain. This adds a function to instantiate the context properly under these assumptions.

- [X] I've added a unit test to test for potential regressions of this bug.
- [x] The changelog has been updated, if applicable.
- [X] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: src/guard/isAccessGrantContext.ts [hunk 1]
```diff
@@ -19,9 +19,9 @@
 // SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
-import { ACCESS_GRANT_CONTEXT } from "../constants";
+import { ACCESS_GRANT_CONTEXT_DEFAULT } from "../constants";
 import { AccessGrantContext } from "../type/AccessGrantContext";
 
 export function isAccessGrantContext(x: unknown): x is AccessGrantContext {
-  return Array.isArray(x) && ACCESS_GRANT_CONTEXT.every((y) => x.includes(y));
+  return Array.isArray(x) && ACCESS_GRANT_CONTEXT_DEFAULT.every((y) => x.includes(y));
```
Comment (matthieubosquet) [inline src/guard/isAccessGrantContext.ts:26]:
```suggestion
  return Array.isArray(x) && x.includes(CONTEXT_VC_W3C);
```

File: src/guard/isAccessGrantContext.ts [hunk 2]
```diff
@@ -19,9 +19,9 @@
 // SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
-import { ACCESS_GRANT_CONTEXT } from "../constants";
+import { ACCESS_GRANT_CONTEXT_DEFAULT } from "../constants";
```
Comment (matthieubosquet) [inline src/guard/isAccessGrantContext.ts:22]:
```suggestion
import { CONTEXT_VC_W3C } from "../constants";
```

File: src/guard/isAccessGrantContext.ts (hunk 1 above)
Comment (matthieubosquet) [reply src/guard/isAccessGrantContext.ts:26]: This ultimately is also used by `getAccessGrant` and `approveAccessRequest` so we can't be too picky in the type guard. I don't think it is worth being more specific for now.

**Review by matthieubosquet — APPROVED** [review]
Just a little issue with the type guard but otherwise looks good! :)

File: src/guard/isAccessGrantContext.ts (hunk 1 above)
Comment (NSeydoux) [reply src/guard/isAccessGrantContext.ts]: Fixed in c186802

**Pull request merged**